### ISSUE TITLE
Mediapool might be able to only initialize one source per type

### ISSUE THREAD

**Issue by alanorozco** [ampproject/amphtml#14058]
Related to #13673.

For each element in the pool an `UpdateSourceTask` is queued during initialization.

Given the [node cloning optimization](https://github.com/ampproject/amphtml/pull/14055/files#diff-76470301c48aae75b698893969623e6cL215), it's possible that this task be scheduled only for the seed element and work for each cloned node.

- [ ] Is this true? If so,
    - [ ] Apply optimization
    - [ ] Reconcile with API refactor in #13673

**Comment by newmuis** [comment]
We'll need to make sure that if it works, it's actually creating separate queues, and not sharing an instance between all of the media elements.

**Comment by alanorozco** [comment]
We can decouple the queuing API from the update source step, no?

I'm not sure we should be updating sources to empty on initialization, as sources could be provided by the component if there is a "zero-time" allocation.

In any case, this is just a nice-to-have and blocked by refactor.

**Comment by newmuis** [comment]
Any videos that don't have a working source at the time we call `blessAll()` don't get blessed 😦 

We could delay their creation until the actual bless call, but that will make the bless operation itself very expensive.

**Comment by ampprojectbot** [comment]
This issue seems to be in  Pending Triage for awhile. @alanorozco Please triage this to an appropriate milestone.

**Comment by ampprojectbot** [comment]
This issue seems to be in  Pending Triage for awhile. @alanorozco Please triage this to an appropriate milestone.

**Comment by alanorozco** [comment]
Sounds like this is overall not worth the effort.
